deployer/templates/hawkular-cassandra-node-pv.yaml

```diff
@@ -28,6 +28,14 @@ parameters:
 - description: The size of storage requested.
   name: PV_SIZE
   value: 10Gi
+- description: Path within the container to the file containing the Certificate Authorities to add to the peer trust store
+  name: TRUSTSTORE_NODES_AUTHORITIES
+  value: '/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt'
+  required: false
+- description: Path within the container to the file containing the Certificate Authorities to add to the client trust store
+  name: TRUSTSTORE_CLIENT_AUTHORITIES
+  value: '/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt'
+  required: false
```

Comment:
We don't want to be accepting any certificate signed by the service-ca certificate, just the Cassandra ones